packages/betterer/src/printer.ts

```diff
@@ -16,8 +17,10 @@ export function print(results: BettererResults): string {
           ? (value as Printable).print()
           : JSON.stringify(value);
     }
-    printedValue = (printedValue as string).replace(/`/g, '\\`');
-    return `\nexports[\`${resultName}\`] = {\n  timestamp: ${timestamp},\n  value: \`${printedValue}\`\n};`;
+    return `\nexports[\`${resultName}\`] = {\n  timestamp: ${timestamp},\n  value: \`${escape(
+      printedValue as string,
+      '"\n'
```

Comment:
Might be good to pull this out to a variable `UNESCAPED` or something to make it clear what it's doing.